feedback.md

```diff
@@ -0,0 +1,14 @@
+# Random Menu
+## What We're Looking For
+
+|  Feature 	|   Feedback	&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|
+|---	|---	|
+|  Random Menu of 10 items displayed in the terminal. 	|   	|
+|  Generator pulls one random item from each array to create menu items. 	|   	|
+|	**Baseline**	|	|
+|  Readable code with consistent indentation. 	|   	|
+|   **Extras**	|   	|
```

Comment:
I wonder if we should keep these only in a "teacher" version of this feedback somehow. I'd really rather not have the students regularly seeing the list of optionals in the feedback since they already have enough negative feelings and confusion about whether or not these things are optional...?